kong/api/app.lua

```diff
@@ -146,8 +147,8 @@ for _, v in ipairs({"kong", "apis", "consumers", "plugins", "cache", "cluster" }
 end
 
 -- Loading plugins routes
-if configuration and configuration.plugins then
-  for _, v in ipairs(configuration.plugins) do
+if singletons.configuration and singletons.configuration.plugins then
```

Comment:
`plugins` is an array of plugin names (strings) used to load the Lua modules. It is built by combining the [bundled plugins](https://github.com/Mashape/kong/blob/master/kong/constants.lua#L31) with the `custom_plugins` configuration property.

`loaded_plugins` is another array, but containing the said plugins *modules* (`handler.lua`) to be executed by the event loop.